ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -471,4 +471,18 @@ VAV_RD_CO2C:
   implements:
   - VAV
   - RD
-  - CO2C
\ No newline at end of file
+  - CO2C
+
+########################
+### Canonical Types  ###
+########################
+
+VAV_SD_DSP_CSOUC:
+  id: "16129160067017605120"
+  description: "Single duct dual set point VAV with cooling control."
+  is_canonical: true
+  implements:
+  - VAV
+  - SD
+  - DSP
```

Comment:
This device will have all of these fields:
- zone_air_temperature_sensor
- zone_air_cooling_temperature_setpoint
- zone_air_heating_temperature_setpoint
- zone_air_unoccupied_cooling_temperature_setpoint
- zone_air_occupied_cooling_temperature_setpoint

Where are the heating setpoints? And why are the unoccupied/occupied setpoints needed if the effective setpoints are already on the device under DSP?